ngraph/core/src/op/sqrt.cpp

```diff
@@ -57,6 +57,7 @@ namespace sqrtop
             NGRAPH_TYPE_CASE(evaluate_sqrt, u64, arg0, out, count);
             NGRAPH_TYPE_CASE(evaluate_sqrt, f16, arg0, out, count);
             NGRAPH_TYPE_CASE(evaluate_sqrt, f32, arg0, out, count);
+            NGRAPH_TYPE_CASE(evaluate_sqrt, f64, arg0, out, count);
```

Comment:
Inference Engine doesn't support `f64` data type. How does it allow to support models?

Reply:
This is just for constant folding. These models have Convert (i64->fp64) -> Sqrt -> Convert (fp64 -> i64).